### PR TITLE
fix: cms blog ui

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

A description of the change.

[AB#1987](https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%206?workitem=1987)

[AB#1986](https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%206?workitem=1986)

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                assert.isOk("blog button exists and active")
+            } else {
+                assert.isNotOk("blog button does not exist")
+            }
+        })
+
+        cy.get("[data-testid=blogs_button]").click()
+        cy.location().should(loc => {
+            expect(loc.pathname).to.eq("/blog")
+        })
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts:38]: Prob should test that there is content being rendered. You can stub the request and return something known. But an integration between us cms and the content being returned.

File: packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        cy.visit("")
+    })
+
+    it("can navigate to /blog page when clicking on the blog button on header", () => {
+        cy.wait("@getStubbedMenu")
+        cy.get("[data-testid=blogs_button]").then($button => {
+            if ($button.is(":visible")) {
+                assert.isOk("blog button exists and active")
+            } else {
+                assert.isNotOk("blog button does not exist")
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts:31]: Should only assert on your expectation. The error from a failed assertion will tell you if you it doesn't exist.

File: packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts (hunk 1 above)
Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts:38]: Added assertion for content on the page.

File: packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    it("can navigate to /blog page when clicking on the blog button on header", () => {
+        cy.wait("@getStubbedMenu")
+        cy.get("[data-testid=blogs_button]").should("exist").click()
+
+        cy.location().should(loc => {
+            expect(loc.pathname).to.eq("/blog")
+        })
+        cy.get("[data-testid=blog_title]")
+            .should("exist")
+            .contains("Yumido Blog")
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts:34]: It's not good practice too assert on text, as text can change. Asserting the presence of the title should be enough to know that the page has loaded. The other thing you could do do is spy to check that it makes the request for the content and that it's resolved.

**Pull request merged**